### ISSUE TITLE
Radio Buttons and Ketchup

### ISSUE THREAD

**Issue by pickleman** [mustardamus/ketchup-plugin#37]
I am having a issue getting Ketchup to validate a set of radio buttons to ensure they made a selection.

There are no examples on the site showing how to handle this.  I have tried the class="validate(required)" but it does not work.  I also tried class="validate(rangeselect(1,3))" and it works but radio buttons are not a multi select thing like checkboxes.

Any assistance would be appreciated.

**Comment by mustardamus** [comment]
Right, radio buttons are left out right now. I'll write an demo later.

**Comment by pickleman** [comment]
Thanks, I have this form which I wish to use Ketchup for but without validating the radio buttons I cannot continue. Is this something you have to write into the JS to handle or is it just a specific way to handle them? 

Also - Is there a way so that the ketchup bubbles are solid (Example: when it hovers over text you cant see the text behind the red bubble?

**Comment by jeffbyrnes** [comment]
I, too, would appreciate some help with validating a radio button set.

**Comment by natoinet** [comment]
Great job! I really like the very neat use of data-validation. 
But, I too, need the radio button.

**Comment by smetal** [comment]
I would love to be able to use this, but i need the support for radio buttons.

**Comment by oalders** [comment]
I also would really like to see an example for radio buttons.

**Comment by paciano** [comment]
@mustardamus -  I think it would be helpful to others if you were to post something on the Ketchup website that states Radio buttons are not supported.
